source/support/decoding-metadata.rst

```diff
@@ -0,0 +1,80 @@
+=================
+Decoding Metadata
+=================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+
+Metadata
+--------
+
+MinIO stores object metadata for erasure coded objects in ``xl.meta`` files.
+
+Each disk in the set containing the object has this file.
+The file is a binary format, requiring a tool to view the values.
+
+You can download the metadata for one or more objects with the MinIO Console's Support > Inspect tool or using :mc-cmd:`mc support inspect` at the command line.
```

Comment:
```suggestion
You can download the metadata for one or more objects with the :guilabel:`MinIO Console's Support > Inspect` tool or using :mc-cmd:`mc support inspect` at the command line.
```

I should probably add labels to the Console page so we can link to these sections as they exist (if such a label does not yet exist)